src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
An alternative might be to go with the `div` implementation by default, but allow the user to pass in a `refToMeasure` instead, in which case `this.props.children` could be rendered directly. 

This is a bit more complex; I'm happy to write it if y'all are okay maintaining it 😉